### DISCUSSION TITLE
Impossible to save sessions

### DISCUSSION THREAD

**Discussion by susancolvine** [sienori/Tab-Session-Manager#863] [Q&A]
Hello! 
For some reasons, Tab Session Manager doesn't react at all whenever I try to save a new session. This issue is has just started today and I don't understand why. 
Help please? 

Update: I've uninstalled and reinstalled TSM, which solved the issue for a while... until it surfaced again, and I'm as lost as ever as I can't find out the cause of the issue. Any kind of tip would be greatly appreciated!

**Comment by susancolvine** [comment]
**Update:** Found a solution to the issue. 
It turns out the issue was due to enabling the "save tab groups" in the options. Disabling this option allowed me to save sessions once again! Therefore I reinstalled "Save tab groups for TSM" extension in the hopes to solve any bug, and the issue seems to have disappeared; I can save sessions with this option enabled... for now. 

**Update II:** The bug happened again. 
In the end, I ended up disabling the "Save tab groups" option permanently, as it keeps causing the bug no matter how many times you reinstall the "Save tab groups for TSM" extension. Hopefully this gets fixed in next update.

**Comment by rediffusion** [comment]
I've tested "Tab Session Manager":

> It was 104 kb after compression of 99,746 kb
> The savings are too small.
